_internal/utils/sourceme.R

```diff
@@ -157,11 +8,15 @@ deploy_all_apps <- function(api_key,
 #' @param username The GitHub username or organization name.
 #' @param repo The name of the repository.
 #' @param ref The reference (commit, branch, or tag) to download from the repository. (default: "HEAD")
+#' @param package_repo The R-Universe mirror where the teal packages would be instealled from.
+#' Main version: https://pharmaverse.r-universe.dev and Release version: https://insightsengineering.r-universe.dev
 #' @param ... Additional arguments passed to `shiny::runApp()`.
 #'
 #' @examples
 #' restore_and_run("app1", username = "insightsengineering", repo = "teal.gallery", ref = "HEAD")
-restore_and_run <- function(app_name, username = "insightsengineering", repo = "teal.gallery", ref = "HEAD", ...) {
+restore_and_run <- function(
+    app_name, username = "insightsengineering", repo = "teal.gallery", ref = "HEAD",
+    package_repo = "https://pharmaverse.r-universe.dev", ...) {
```

Comment:
it's better to have ref to stable as the default value

Reply:
Sure, after the release it will be changed. Right now it was like this because the stable release apps are broken. After the release, we will create a separate branch to test the main branch to make sure that both deployments have their own branches to be safe from breaking changes.